person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
-
+# base class for student and teache
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job fixing all changes mentioned by the previous reviewer. However I think it would be better if you update your `attr_accessor` values by adding `rentals` and remove it from the `attr_reader` as it is required for the project that `Getters` and `Setters` have to be present for `@rentals`.